### PR TITLE
Update Passage-Ruby With New Readme and License

### PR DESCRIPTION
**Pull request by flanagankp**
## What's New?

Update to the new README template, updated license, updated metadata in gemspec file.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added/updated inline documentation for public facing interfaces if relevant
- [x] My changes generate no new warnings

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,218 +1,84 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+![passage-ruby](https://storage.googleapis.com/passage-docs/github-md-assets/passage-ruby.png)
 
-[![Gem Version](https://badge.fury.io/rb/passageidentity.svg)](https://badge.fury.io/rb/passageidentity)
+![Gem Version](https://img.shields.io/gem/v/passageidentity)
```
Comment (jenn-macfarlane) [inline README.md:3]: I think we're putting these shields on a single line (or at least I have been!)

**Review by jenn-macfarlane — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (flanagankp) [reply README.md:3]: I have been as well - if you're looking at the preview you're probably seeing the old badge.fury shield I removed above the news

Comment (flanagankp) [reply README.md:3]: Top one is showing what was removed, the bottom three are the actual ones that will be there: 
image

Comment (jenn-macfarlane) [reply README.md:3]: Whoops you're right, didn't realize I was in unified view!

**Pull request merged**